virocon/distributions.py

```diff
@@ -861,7 +861,7 @@ def _fit_mle(self, sample):
         if self.f_sigma is not None:
             fparams["fscale"] = self.f_sigma
 
-        self.mu, self.sigma = sts.lognorm.fit(
+        self.mu, self.sigma = sts.norm.fit(
```

Comment:
This is obviously a bug. Thanks for finding it. We should write a simple test, that fails for this bug and passes with your correction.

Reply:
We decided to leave this kind of bug untested for now. We will implement tests at a later point in time.